test/mocha/new.js

```diff
@@ -2,7 +2,7 @@ var assert = require("assert");
 var uglify = require("../../");
 
 describe("New", function() {
-    it("Should attach callback parens after some tokens", function() {
+    it("Should attach callback parentheses in beautify mode", function() {
```

Comment:
What does "attach callback" mean? Why not "Should add parentheses for `new` expressions in beautify mode?"